src/Microsoft.Health.Dicom.Api/Features/Routing/VersionedRouteAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.Health.Dicom.Api.Features.Routing
+{
+    public sealed class VersionedRouteAttribute : RouteAttribute
+    {
+        public VersionedRouteAttribute(string template) : base("v{version:apiVersion}/" + template)
```

Comment:
nit: I think the style in the repository is to place the nested ctor on the next line and indented:
```csharp
public VersionedRouteAttribute(string template)
    : base("v{version:apiVersion}/" + template)
```

Reply:
Thanks! Updated